### PR TITLE
Fix a few mistakes I made

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
from #82683?

**Comment by optimumtact** [comment]
yes

File: SQL/tgstation_schema_prefixed.sql [hunk 1]
```diff
@@ -214,6 +214,23 @@ CREATE TABLE `SS13_ipintel` (
...
+CREATE TABLE `SS13_ipintel_whitelist` (
+	`ckey` varchar(32) NOT NULL,
+	`admin_ckey` varchar(32) NOT NULL,
+	PRIMARY KEY (`ckey`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `legacy_population`
+--
```
Comment (Ghommie) [inline SQL/tgstation_schema_prefixed.sql:233]:
```suggestion
```
dupe

File: SQL/database_changelog.md [hunk 2]
```diff
@@ -16,19 +16,22 @@ INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 27);
...
+	`ckey` varchar(32) NOT NULL,
+	`admin_ckey` varchar(32) NOT NULL,
+	PRIMARY KEY (`ckey`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `legacy_population`
+--
+
```
Comment (Ghommie) [inline SQL/database_changelog.md:34]:
```suggestion
```
ditto

**Pull request merged**